hack/kubestellar.sh

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env bash
```

Comment:
This is a command that _users_ use, so they should get it in `bin`.  Being a script, the source should be in `scripts/`; the `hack/` directory is for scripts that are specifically aimed at _contributors_.

Reply:
Fixed